### PR TITLE
Style bugs fixed: cards, topic and tag pages

### PR DESCRIPTION
**Pull request by UladBohdan**
* Long names on Person Tag Cards
* Columns on Topic Page
* Cards hover based on device being able to hover; not by screen size
* Tag Page minor fixes (extra line removed; title fixed)

Some scss refactoring done.

### CODE REVIEW

**Comment by UladBohdan** [comment]
[Article & Tag Cards](https://trello.com/c/K2rdecIj/427-article-tag-cards)

**Comment by UladBohdan** [comment]
[Topic Page](https://trello.com/c/wp2cnZ7W/419-topic-page)

**Comment by UladBohdan** [comment]
[hover for cards on touch vs. desktop devices](https://trello.com/c/plM7kzAX/430-hover-for-cards-on-touch-vs-desktop-devices)

**Comment by UladBohdan** [comment]
[Tag Page](https://trello.com/c/akYk0eJ6/425-tag-page)

File: components/articles/blocks/BlockWrapper.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import cn from 'classnames';
+
+const BlockWrapper = ({ className, withBackground, children }) => (
+  <div
+    className={cn(
+      'wir-content-padding block',
+      { 'block__with-background': withBackground, 'block__no-background': !withBackground },
```
Comment (Drapegnik) [inline components/articles/blocks/BlockWrapper.js]: maybe:
```js
`block__${withBackground ? 'with' : 'no'}-background`
```

File: pages/tag.js [hunk 2]
```diff
@@ -46,6 +46,10 @@ const ArticlesBlocks = ({ articlesCount, blocks }) => {
   }
 
   return blocks.map((block, index) => {
+    if (!block.length) {
+      // Q: How's that happening last block always contains no articles?
+      return <></>;
```
Comment (Drapegnik) [inline pages/tag.js]:
```suggestion
      return null;
```

File: utils/getters.js [hunk 3]
```diff
@@ -101,10 +101,13 @@ export const getLocalizedTeam = localizeArray(localizeFields(['name', 'role']));
 
 export const getLocalizedVacancies = localizeArray(localizeFields(['title', 'description']));
 
-// returns list of articles coupled by 2 or 3
+// returns a list of articles coupled by 2 or 3
 // 5 articles = 2 blocks (2 + 3)
 export const getArticlesBlocks = articles =>
-  chunk(articles, 5).reduce(
-    (blocks, blockOf5) => blocks.concat([blockOf5.slice(0, 2), blockOf5.slice(2)]),
```
Comment (Drapegnik) [inline utils/getters.js:108]:
```js
[blockOf5.slice(0, 2), blockOf5.slice(2)].filter(b => b.length)
```

**Review by Drapegnik — APPROVED** [review]

**Pull request merged**